config/config.yaml

```diff
@@ -118,8 +129,9 @@ params:
   # also, separate capitalised letter flags are required for adapters in
   # the reverse reads of paired end sequencing:
   # * https://cutadapt.readthedocs.io/en/stable/guide.html#trimming-paired-end-reads
-  cutadapt-se: ""
+  cutadapt-se: " "
   # reasoning behind parameters:
+  #   For reads that are produced by 3’-end sequencing first 12 bases are removed based on accordance with Lexogen’s recommendation. More info: https://www.nature.com/articles/s41598-019-55434-x#Sec10
```

Comment:
```suggestion
  #   For reads that are produced by 3’-end sequencing, depending on the protocol, it might be recommended to remove some leading bases (e.g. see https://www.nature.com/articles/s41598-019-55434-x#Sec10)
```

Reply:
I think this comment (with my changes) should be moved into the cutadapt-se part above.